src/test/java/hudson/plugin/versioncolumn/JVMVersionComparatorTest.java

```diff
@@ -62,17 +66,19 @@ private JVMVersionComparator.ComparisonMode toExactMatch(boolean exactMatch) {
 
     private static Object[] parametersForCompatibleBytecodeLevel() {
         return new Object[][]{
-                {"1.8.0", JVMConstants.JAVA_8},
-                {"1.8.0", JVMConstants.JAVA_7},
-                {"1.8.0", JVMConstants.JAVA_6},
+                {"1.8.0", JavaSpecificationVersion.JAVA_8.toClassVersion()},
+                {"1.8.0", JavaSpecificationVersion.JAVA_7.toClassVersion()},
+                {"1.8.0", JavaSpecificationVersion.JAVA_6.toReleaseVersion()},
+                {"11.0.1", JavaSpecificationVersion.JAVA_11.toReleaseVersion()},
```

Comment:
Why aren't these `toClassVersion`? This parameter represents a class version, not a release version.